Importer/ImporterInterface.php

```diff
@@ -9,18 +9,16 @@
 interface ImporterInterface
 {
     /**
-     * @param array $data
-     *
-     * @return object[]
+     * @param string $csv
+     * @param bool   $dryRun
      */
-    public function import($data);
+    public function import($csv, $dryRun);
```

Comment:
It doesn't really make sense for this `$csv` parameter to be here.
Maybe this method shouldn't even be here.